advanced_functionality/data_distribution_types/data_distribution_types.ipynb

```diff
@@ -39,20 +39,34 @@
     "- The IAM role arn used to give training and hosting access to your data. See the documentation for how to create these.  Note, if more than one role is required for notebook instances, training, and/or hosting, please replace the boto regexp with a the appropriate full IAM role arn string(s)."
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    &nbsp;# bucket = "&lt;your_s3_bucket_name_here&gt;"
</pre><p>Thanks for switching to the default bucket. Please go ahead and delete the commented-out "your s3 bucket" line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3284/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done!